### DISCUSSION TITLE
Change log for deleted/create cables doesn't show termination points.

### DISCUSSION THREAD

**Discussion by DenHayes** [netbox-community/netbox#10779] [Q&A]
We had a user accidentially delete a cable in Netbox and we are attempting to rebuild it.  Unfortunately,  the change log entry for the delete action doesn't capture the termination information for the cable.

![image](https://user-images.githubusercontent.com/22125585/198389607-f48604eb-2a14-4fc7-b4a6-cfeadec08ddf.png)

The create change log entry doesn't show the information either and the object is deleted, so there's no hyperlink to see the info.

Is there a way to see the termination end points for this cable, so we can manually re-create it?

**Comment by drmsoffall** [comment]
The changelog should capture all related changes for a given request, including changes to interfaces/terminations.  These should be near this change in the full changelog view.  You could also copy and paste the Request ID from the single change view into a URL like this:

http://your.netbox.server/extras/changelog/?request_id=94b4...etc

This will show all the related changes, and you should be able to gather enough info to recreate it.

**Reply by DenHayes** [reply]
Unfortunatly, I'm not seeing any termination info when I look at either the create or delete detailed change log.

![image](https://user-images.githubusercontent.com/22125585/198673394-26481bf3-c57c-4b8a-a5a1-4f9e9d3cf346.png)

![image](https://user-images.githubusercontent.com/22125585/198673095-450b13b6-afc0-4bcd-9991-64bf81db242c.png)

Maybe a bug? I'm running version 3.3.4.

**Comment by mtinberg** [comment]
This may seem like a simple thing, but you need to click on the change log entry to view the details (before/after diffs and such).


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________

**Reply by DenHayes** [reply]
I've clicked on the ID and can see the pre and post data, but this data only contains the characteristics of the cable; see my first post on what I see. 

I've also clicked on the Request ID and don't see what I'm looking for either.

**Comment by mtinberg** [comment]
Sorry for my poor following along, I didn't refer to your original screenshot.  In my Netbox 3.2.9 instance I see the termination points on deleted cables, which are missing in your original report.

[cid:cdbdd650-9e5a-48b0-acb1-06cf81f5f9ff]

[cid:8f8fb64b-b387-470b-82df-021ea25062a6]


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________

**Reply by DenHayes** [reply]
We upgraded our lab to the latest, v3.3.6, and we're still not seeing the termination information when deleting a cable.  I believe this to be a bug, since you're seeing it in your version; I'll open a issue.

**Comment by mtinberg** [comment]
That tickled a memory

https://github.com/netbox-community/netbox/releases/tag/v3.3.0

The cable model was changed for 3.3.x (which is why I stayed on 3.2.9 until I get around to updating my API clients, I'd just backburnered the work and forgot the details), is it possible that the changelog you seek is filed under dcim.cable-terminations and not dcim.cable

I tried in the demo environment to add an interface to termination b

https://demo.netbox.dev/dcim/cables/1/

and looked for a changelog entry for dcim.cable-terminations but didn't see one

https://demo.netbox.dev/extras/changelog/?changed_object_type_id=59

The only object change is for dcim.cable and doesn't include the termination data

https://demo.netbox.dev/extras/changelog/212/

—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________

**Reply by DenHayes** [reply]
I'm still not seeing anything I can use to reconstruct a cable connection.  On the odd occasion I'll see a seperate change log entries for interface changes updating their status, but it's not consistant and can't rely on them.  

I would hope the change log would capture the termination info too.
